src/main/java/uk/gov/hmcts/darts/common/datamanagement/component/impl/DownloadResponseMetaData.java

```diff
@@ -18,27 +20,22 @@ public abstract class DownloadResponseMetaData implements Closeable {
     protected OutputStream outputStream;
 
     @Getter
-    private boolean processedByContainer;
-
-    @Getter
-    private boolean successfulDownload;
+    @Setter
+    private ExternalObjectDirectoryEntity eodEntity;
 
     @Getter
+    @Setter
     private DatastoreContainerType containerTypeUsedToDownload;
 
     public void markSuccess(DatastoreContainerType containerType) {
```

Comment:
Not sure if something else needs to happen in these markSuccess / markFailure methods?
Can you check please - I guess you are logging exceptions elsewhere, wondering if they're even needed now

Reply:
i've now removed it.